modules/apple_support/1.3.1/patches/skip-objc_library-target-on-non-mac-platforms.patch

```diff
@@ -0,0 +1,26 @@
+From 9fd8799b3e1641b590cf93a51fd475b7a792f350 Mon Sep 17 00:00:00 2001
+From: Patrick Balestra <me@patrickbalestra.com>
+Date: Tue, 27 Sep 2022 10:27:33 -0700
+Subject: [PATCH] Skip objc_library target on non-macos platforms
+
+---
+ lib/BUILD | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/lib/BUILD b/lib/BUILD
+index 024dc9b..715c59a 100644
+--- a/lib/BUILD
++++ b/lib/BUILD
+@@ -13,6 +13,9 @@ objc_library(
+     srcs = ["swizzle_absolute_xcttestsourcelocation.m"],
+     visibility = ["//visibility:public"],
+     alwayslink = True,
++    target_compatible_with = [
++        "@platforms//os:macos",
++    ],
```

Comment:
We can just add this to `apple_support`, right?

Reply:
I'm planning on doing that, I wanted to test out if I could make this work with a patch for this specific release first.

Reply:
Got it to work just now, so yes this fixes it. What do you think of merging this for 1.3.1 and I'll immediately fix this directly in `apple_support` so the following release will remove the need for the patch?

Reply:
No opinion on that.